cpp/examples/abm_minimal.cpp

```diff
@@ -142,7 +142,7 @@ int main()
     for (auto& person : persons) {
         mio::abm::InfectionState infection_state =
             (mio::abm::InfectionState)(rand() % ((uint32_t)mio::abm::InfectionState::Count - 1));
-        if (infection_state != mio::abm::InfectionState::Susceptible) {
+        if (infection_state != mio::abm::InfectionState::Susceptible)
```

Comment:
The removal of one opening curly bracket looks wrong. Are you sure this is correct?

Also for one-line `if`s, please use `{` and `}`